frontend/src/utils/index.js

```diff
@@ -311,7 +314,9 @@ export function getProjectDetails (project) {
     description,
     purpose,
     staleSinceTimestamp,
-    staleAutoDeleteTimestamp
+    staleAutoDeleteTimestamp,
+    phase,
+    isNotReady
```

Comment:
If we need this boolean property I would use a getter in this case.
```suggestion
    get isNotReady () { return this.phase !== 'Ready' }
```